Tests/Decoda/DecodaTest.php

```diff
@@ -4,13 +4,8 @@
 
 use FM\BbcodeBundle\Decoda\Decoda;
 
-class DecodaTest extends \PHPUnit_Framework_TestCase
+class DecodaTest extends \PHPUnit\Framework\TestCase
 {
-    public function testConstructor()
-    {
-        $result = new Decoda();
-    }
-
```

Comment:
What is the reason to delete this case?

The assertion here is that no exception is thrown.

Reply:
You are not testing anything, since the constructor will never throw an error. In fact, why is this bundle even testing third party code?